admin/data/sql/migrations/20190528131913.do.remove-checkdata-from-vewcompletedcheckcount.sql

```diff
@@ -0,0 +1,6 @@
+ALTER VIEW [mtc_admin].[vewCompletedCheckCount] AS
+  SELECT c.pupil_id, COUNT(c.id) AS [CompletedCheckCount]
```

Comment:
This appears to be the number of pupils with completed checks.  Perhaps rename the alias for clarity?